### ISSUE TITLE
Some content on the website seems a bit out of date

### ISSUE THREAD

**Issue by steverao** [open-telemetry/opentelemetry.io#4092]
**What happened?** Describe the problem that occurred.

I found [Suppressing specific agent instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) in website is inconsistent with the latest [supported-libraries](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md).
There are some problems I think maybe we need to solve them:
1. There are some new libraries such as `Alibaba Druid` and `MyBatis` in supported-libraries, but they can't find in Suppressing specific agent instrumentation.
2. Some librarie's name are also inconsistent on both sides. Such as in website, Cassandra's name is `Apache Cassandra`, but its name is `Cassandra Driver` in supported-libraries.

**What did you expect would happen?** Describe the expected result/output.

How to update the website document before? Whether we should come up with a method ensure that both parties can be consistent in a timely so that similar problems will not continue to occur？

**Comment by svrnm** [comment]
@open-telemetry/java-instrumentation-approvers @open-telemetry/java-approvers please take a look

**Comment by breedx-splk** [comment]
@open-telemetry/java-instrumentation-approvers did we auto-generate that list once upon a time? I'm guessing it wasn't hand edited.....was it?

**Comment by jeanbisutti** [comment]
[supported-libraries](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md) seems hand edited and should probably be migrated to the https://opentelemetry.io/ website.

@steverao Don't hesitate to create PRs to improve the documentation and suggests automatic checks if it would be possible without too much effort.

**Comment by steverao** [comment]
> supported-libraries

I think whether we don't provide the table here in the future, and add instrumentation name in [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks). The final result is similar to the picture below:
![image](https://github.com/open-telemetry/opentelemetry.io/assets/25423978/003960cc-ff22-44cd-929c-60ee27668f3c)
In this way, the content is relatively concentrated and easy to maintain. All other places that involve relevant content can be quoted directly. Such as [enable-only-specific-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-only-specific-instrumentation), [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) and [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations). 
@open-telemetry/java-instrumentation-approvers What do you think or do you have any better suggestions?

**Comment by jeanbisutti** [comment]
> I think whether we don't provide the table here in the future, and add instrumentation name in [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks). The final result is similar to the picture below: ![image](https://private-user-images.githubusercontent.com/25423978/310519594-003960cc-ff22-44cd-929c-60ee27668f3c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MzQzNjgsIm5iZiI6MTcwOTczNDA2OCwicGF0aCI6Ii8yNTQyMzk3OC8zMTA1MTk1OTQtMDAzOTYwY2MtZmYyMi00NGNkLTkyOWMtNjBlZTI3NjY4ZjNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA2VDE0MDc0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmNzNjYzUyYWU3YjE3NWQyNjE1NWY0MjQ3OTczYzE1ZGEyNDI2M2NhM2Y2ZGU5MzdhNjJhZDNhYjNkYzg0ZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.32fU-hzb0Z9Bd4fE_3h5trT6JAlcy7b0IWbWdIvUljw) In this way, the content is relatively concentrated and easy to maintain. All other places that involve relevant content can be quoted directly. Such as [enable-only-specific-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-only-specific-instrumentation), [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) and [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations). @open-telemetry/java-instrumentation-approvers What do you think or do you have any better suggestions?

@steverao You seem to suggest to add a new column with the automatic instrumentation name to the instrumentation libraries table.

Generally, if a user has configured the OpenTelemetry Java agent, then he does not need instrumentation libraries. So, it may make sense to document the instrumentation of the Java agent and the instrumentation libraries on different pages, and to create a new page for the instrumentation libraries on the https://opentelemetry.io/ website.

> All other places that involve relevant content can be quoted directly. Such as [enable-only-specific-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-only-specific-instrumentation), [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) and [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations).

[disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations) does not seem documented on the https://opentelemetry.io/ website. The https://opentelemetry.io/docs/languages/java/automatic/configuration/ should probably be updated.

**Comment by jeanbisutti** [comment]
In addition, I would sugest to add the framework links  and a `Semantic Conventions` column to the [this table](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation).

@steverao and @open-telemetry/java-instrumentation-approvers WDYT?

**Comment by steverao** [comment]
> > I think whether we don't provide the table here in the future, and add instrumentation name in [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks). The final result is similar to the picture below: ![image](https://private-user-images.githubusercontent.com/25423978/310519594-003960cc-ff22-44cd-929c-60ee27668f3c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MzQzNjgsIm5iZiI6MTcwOTczNDA2OCwicGF0aCI6Ii8yNTQyMzk3OC8zMTA1MTk1OTQtMDAzOTYwY2MtZmYyMi00NGNkLTkyOWMtNjBlZTI3NjY4ZjNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA2VDE0MDc0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNmNzNjYzUyYWU3YjE3NWQyNjE1NWY0MjQ3OTczYzE1ZGEyNDI2M2NhM2Y2ZGU5MzdhNjJhZDNhYjNkYzg0ZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.32fU-hzb0Z9Bd4fE_3h5trT6JAlcy7b0IWbWdIvUljw) In this way, the content is relatively concentrated and easy to maintain. All other places that involve relevant content can be quoted directly. Such as [enable-only-specific-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-only-specific-instrumentation), [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) and [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations). @open-telemetry/java-instrumentation-approvers What do you think or do you have any better suggestions?
> 
> @steverao You seem to suggest to add a new column with the automatic instrumentation name to the instrumentation libraries table.
> 
> Generally, if a user has configured the OpenTelemetry Java agent, then he does not need instrumentation libraries. So, it may make sense to document the instrumentation of the Java agent and the instrumentation libraries on different pages, and to create a new page for the instrumentation libraries on the https://opentelemetry.io/ website.
> 
> > All other places that involve relevant content can be quoted directly. Such as [enable-only-specific-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-only-specific-instrumentation), [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation) and [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations).
> 
> [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations) does not seem documented on the https://opentelemetry.io/ website. The https://opentelemetry.io/docs/languages/java/automatic/configuration/ should probably be updated.

Yes, my idea is to add new column in [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks). Just maintain one table, it won't miss to update information when someone adds a new instrumentation to code repo(Solution 1). 
Your thought about to create a new table in website and list the instrumentation of the Java agent and the instrumentation libraries on different pages(Solution 2). I think it's another way to optimize the current situation. 
The current approach only lists the `[name]` table in [suppressing-specific-agent-instrumentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation), and there is no clear explanation of the context of other places where `[name]` is mentioned, which is not particularly user-friendly. 
As mentioned by jean, If we choose solution 2, maybe we can edit a new table in other places on the website, besides instrumentation `[name]` and we can add some other columns, such as framework links and semantic conventions. If we choose this way, It will make maintenance a little more complicated, but I think it is acceptable. We can pay attention to it when merging new instrumentation code later.
Finally for [disabled-instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations). At first, I think we just list the framework according the [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks) instead of instrumentation name, and put the `[name]` table link,  users can know how to enable an instrumentation or disable one.

**Comment by svrnm** [comment]
> Yes, my idea is to add new column in [supported tables](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks). Just maintain one table, it won't miss to update information when someone adds a new instrumentation to code repo(Solution 1).

Just to throw this in as a perspective from docs: We should aim to have material on opentelemetry.io to be the source of truth, and (if possible) not the repositories, especially for those end-user facing details. We have way to many documentation fragmented across the individual repositories making it really hard for end-users to find the things they need.

If this is a workflow issues (_"it won't miss to update information when someone adds a new instrumentation to code repo"_), we should address this as such and collaborate on a solution. This can reach from a metadata file stored in the repository that we pull from OR (my preferred solution) this details become part of the registry entries per instrumentation library (see https://github.com/open-telemetry/opentelemetry.io/blob/main/data/registry/instrumentation-java-apache-httpclient.yml and https://opentelemetry.io/ecosystem/registry/?language=java&component=instrumentation), which eventually would us even allow to make this data searchable via the Registry.

**Comment by cartermp** [comment]
Yep, I think there's a technical solution we can employ to keep stuff always up to date. Copying data from the repos got us pretty far for now but for stuff like this we should use a path that's largely automated by the bots.

**Comment by jeanbisutti** [comment]
> Just to throw this in as a perspective from docs: We should aim to have material on opentelemetry.io to be the source of truth, and (if possible) not the repositories

So, it seems also to make sense for the doc approvers to move the [list of instrumentation libraries](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks) to the website. It could be done with one PR. The documentation of each instrumenation libraries could perhaps be moved with other PRs.

The next question seems to be:  _should we have one table for the [instrumentations of the Java agent](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-manual-instrumentation-only) and another one for the instrumentation libraries, or only one table_?

As I mentioned before, I would prefer two tables from a user perspective:

>Generally, if a user has configured the OpenTelemetry Java agent, then he does not need instrumentation libraries. So, it may make sense to document the instrumentation of the Java agent and the instrumentation libraries on different pages, and to create a new page for the instrumentation libraries on the https://opentelemetry.io/ website.

I will add the question about one or two tables to the agenda of the OTel Java SIG today.

Today, the Java agent has many instrumentations ar there are many instrumentation libraries. It seems to have became not frequent to add a new instrumentation to the Java agent or a new instrumentation library. So, about the documentation maintenance, I would suggest to fix the current inconsistencies (see https://github.com/open-telemetry/opentelemetry.io/issues/4092#issue-2167107404), and  and to see over time if other inconsistencies are frequently introduced.

**Comment by jeanbisutti** [comment]
@steverao The topic has been discussed in the last Java SIG. It has been suggested to:
 * Move the documentation of the Java instrumentation libraries to https://opentelemetry.io/
 * Don't mix the automatic instrumentations of the OpenTelemetry Java agent with the Java instrumentation libraries
 * Use metadata files in the long term

@open-telemetry/docs-approvers Where would you recommend placing the table of the Java instrumentation libraries on the website?

**Comment by jeanbisutti** [comment]
To sum up the shorty-term improvements discussed:
* >There are some new libraries such as Alibaba Druid and MyBatis in supported-libraries, but they can't find in Suppressing specific agent instrumentation.
*  > Some librarie's name are also inconsistent on both sides. Such as in website, Cassandra's name is Apache Cassandra, but its name is Cassandra Driver in supported-libraries.
* [Table of the automatic instrumentations](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation): 
> add some other columns, such as framework links and semantic conventions
* Move the table of the Java instrumentation libraries to the website (waiting for a response to https://github.com/open-telemetry/opentelemetry.io/issues/4092#issuecomment-1988171886)

@steverao  If you would be interested in creating doc PRs to address these points, that would be great contributions useful for the OpenTelemetry users.

**Comment by steverao** [comment]
> To sum up the shorty-term improvements discussed:
> 
> * > There are some new libraries such as Alibaba Druid and MyBatis in supported-libraries, but they can't find in Suppressing specific agent instrumentation.
> * > Some librarie's name are also inconsistent on both sides. Such as in website, Cassandra's name is Apache Cassandra, but its name is Cassandra Driver in supported-libraries.
> * [Table of the automatic instrumentations](https://opentelemetry.io/docs/languages/java/automatic/configuration/#suppressing-specific-agent-instrumentation):
> 
> > add some other columns, such as framework links and semantic conventions
> 
> * Move the table of the Java instrumentation libraries to the website (waiting for a response to [Some content on the website seems a bit out of date #4092 (comment)](https://github.com/open-telemetry/opentelemetry.io/issues/4092#issuecomment-1988171886))
> 
> @steverao If you would be interested in creating doc PRs to address these points, that would be great contributions useful for the OpenTelemetry users.

The time of Java SIG meeting starts at one o'clock in the morning in my country, so It's hard for me to attend the last meeting：( I saw the replay and agree with that decision. Please assign the issue to me, I will finish it in this week.

**Comment by jeanbisutti** [comment]
@steverao Indeed, it's very late for you. Thank you for your help!

**Comment by theletterf** [comment]
A related suggestion: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24189

**Comment by svrnm** [comment]
I suspect this will be covered by the explorer work? cc @jaydeluca
